.github/workflows/pulsar-ci.yaml

```diff
@@ -648,6 +651,18 @@ jobs:
         run: |
           $GITHUB_WORKSPACE/build/pulsar_ci_tool.sh docker_load_image_from_github_actions_artifacts pulsar-java-test-image
 
+      - name: Pull apachepulsar/pulsar:2.10.6
+        run: |
+          docker pull apachepulsar/pulsar:2.10.6
+
+      - name: Pull apachepulsar/pulsar:3.0.5
+        run: |
+          docker pull apachepulsar/pulsar:3.0.5
+
+      - name: Pull alpine:3.20.1
+        run: |
+          docker pull alpine:3.20.1
+
```

Comment:
This is unnecessary and additional overhead for most builds in this matrix. TestContainers will pull the images when it's necessary to do that.